microservices/users/src/repositories/user.ts

```diff
@@ -99,6 +100,71 @@ class User extends Repository<UserEntity> {
       status: 403,
     });
   }
+
+  /**
+   * Clear  user tokens
+   * @description If token passed - clear rest user tokens
+   * If token not passed - clear all user tokens
+   */
+  public async clearUserTokens(userId: string, tokenId?: string): Promise<void> {
+    const query = {
+      where: {
+        userId,
+        ...(tokenId ? { id: tokenId } : {}),
+      },
+    };
+
+    const { error: countError, result: countResult } = await Api.get().authentication.token.count({
+      query,
+    });
+
+    if (countError) {
+      Log.error(countError.message);
+
+      throw new BaseException({
+        status: 500,
+        message: 'Failed to clear rest user tokens.',
+        payload: {
+          userId,
+          currentToken: tokenId,
```

Comment:
I'm told you about message! countError.message

Reply:
I got it